source/_patterns/02-molecules/vue-widgets/demo/index.js

```diff
@@ -32,6 +33,14 @@ if (document.getElementById('vue-html-example-2')) {
     }))();
 }
 
+if (document.getElementById('vue-style-example')) {
```

Comment:
I'll rename these as well. I did not change these files yet.

Reply:
This file is actually not needed.

Reply:
This file is needed.

Reply:
I could write another component folder and move this into it's own part of the widget SRC. It is not like the other components because the format is simpler.

Reply:
The `index.js` files within all `demo/` folders are required for Webpack to find and auto-reload changes within `demo/`.